util/missingmacro.txt

```diff
@@ -194,7 +194,9 @@ X509_extract_key
 X509_REQ_extract_key
 X509_name_cmp
 X509_LOOKUP_load_file
+X509_LOOKUP_load_store
 X509_LOOKUP_add_dir
+X509_LOOKUP_add_store
```

Comment:
Errr....no!! Please document these things

Reply:
In a separate PR.  If it was *only* the store function, then I would add it here, but it looks like that whole set of functions needs to be documented, and I find that off-topic for this PR.

So, if I promise to do so, will that be satisfactory?

Reply:
Ok...but I will check!!

Reply:
So....I checked.....

Ping @levitte

Reply:
Meh...  I was sure I wrote something...  But in all the piles of branches I handle, it may be that it got lost.

Reply:
See #10986